components/release/development/kustomization.yaml

```diff
@@ -2,11 +2,12 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/release-service/config/default?ref=0e7c27c9e6af34f26910985c0d057ff76ba9243c
+  - https://github.com/redhat-appstudio/release-service/config/default?ref=8a65c03f99cc3190f280b3fd1ec1f400095be5e7 
+
```

Comment:
@happybhati I don't think this extra newline should be here. It must be coming from the script in the RPA

Reply:
@johnbieren I am wondering why it added new line to only this file not the other one?
Does anything catch your eye? 

Ref: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/216/diffs

Reply:
CC: @mmalina 
Please take a look https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/217/diffs#9ec29cd0b2d87ab0786d0ecefa69cfe0f688349c

I hope this will solve the issue but you are expert so need you to give it a look.

Reply:
I added a comment there: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/217/diffs#note_9995483

It seems that when you use `\+newline` inside a sed expression it will translate as a new line (even if normally it should be ignored). So if I move the line break to be between expressions, it works correctly for me.